zerver/actions/invites.py

```diff
@@ -238,20 +238,20 @@ def do_invite_users(
         if email_error:
             errors.append((email, email_error, False))
         else:
-            good_emails.add(email)
+            good_emails.add((email, full_name))
 
     """
     good_emails are emails that look ok so far,
     but we still need to make sure they're not
     gonna conflict with existing users
     """
-    error_dict = get_existing_user_errors(realm, good_emails)
+    error_dict = get_existing_user_errors(realm, {email for email, _ in good_emails})
```

Comment:
`_` is not a good name for such a variable in a Django project; we usually use `ignored` here.